Can't build seqan with standard Makefile (‘directionIterator’ was not declared in this scope)

I tried to create a simple "Hello seqan" project, following this tutorial: http://seqan.readthedocs.org/en/master/BuildManual/IntegrationWithYourOwnBuildSystem.html#build-manual-integration-with-your-own-build-system

However, without success. I also tried to compile the <b>util/makefile_project</b> project - no success again.

> $ uname -a
Linux RD-SD-29-KIRIL 3.13.0-45-generic #74-Ubuntu SMP Tue Jan 13 19:36:28 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

> $ g++ --version
g++ (Ubuntu 4.8.2-19ubuntu1) 4.8.2

> $ make
make -C debug
make[1]: Entering directory `xxxxx/seqan-src-tmp/util/makefile_project/debug'
g++ -I../../../include -I../../../include -g -O0 -DSEQAN_ENABLE_TESTING=0 -DSEQAN_ENABLE_DEBUG=1 -c -o main.o ../src/main.cpp
In file included from ../../../include/seqan/sequence.h:105:0,
                 from ../src/main.cpp:4:
../../../include/seqan/sequence/string_base.h: In instantiation of ‘TStream& seqan::operator<<(TStream&, const seqan::String<TRightValue, TRightSpec>&) [with TStream = std::basic_ostream<char>; TValue = char; TSpec = seqan::Alloc<>]’:
../src/main.cpp:10:40:   required from here
../../../include/seqan/sequence/string_base.h:2009:94: error: ‘directionIterator’ was not declared in this scope
     typename DirectionIterator<TStream, Output>::Type it = directionIterator(target, Output());
                                                                                              ^
make[1]: *** [main.o] Error 1
make[1]: Leaving directory `xxxxx/seqan-src-tmp/util/makefile_project/debug'
make: *** [debug] Error 2

Building and running demo projects with <b>cmake</b> works fine (also tried the same "Hello seqan" project, but with <b>cmake</b> - no issues at all).

--- a/util/makefile_project/src/main.cpp
+++ b/util/makefile_project/src/main.cpp
@@ -1,12 +1,11 @@
-#include <iostream>
-
 #include <seqan/basic.h>
 #include <seqan/sequence.h>
-#include <seqan/file.h>
+#include <seqan/stream.h>
 
 using namespace seqan;
 
-int main() {
-	std::cout << CharString("Hello SeqAn!") << std::endl;
+int main()
+{
+    std::cout << CharString("Hello SeqAn!") << std::endl;
     return 0;
 }

